edx_event_bus_redis/__init__.py

```diff
@@ -5,6 +5,4 @@
 from edx_event_bus_redis.internal.consumer import RedisEventConsumer
 from edx_event_bus_redis.internal.producer import create_producer
 
-__version__ = '0.5.1'
-
-default_app_config = 'edx_event_bus_redis.apps.EdxEventBusRedisConfig'  # pylint: disable=invalid-name
```

Comment:
This line was removed by django-upgrade. See:

https://github.com/adamchainz/django-upgrade/blob/main/src/django_upgrade/fixers/default_app_config.py
https://docs.djangoproject.com/en/stable/releases/3.2/#features-deprecated-in-3-2